Cargo.toml

```diff
@@ -10,23 +10,23 @@ license = "IJG"
 name = "mozjpeg"
 readme = "README.md"
 repository = "https://github.com/ImageOptim/mozjpeg-rust"
-version = "0.9.1"
+version = "0.9.2"
 edition = "2018"
 
 [dependencies]
-libc = "0.2.98"
-mozjpeg-sys = { version = "1.0.0", default-features = false }
-rgb = { version = "0.8.27", features = ["as-bytes"] }
-arrayvec = "0.7.1"
-fallible_collections = "0.4.2"
+libc = "0.2"
+mozjpeg-sys = { version = "1.0", default-features = false }
+rgb = { version = "0.8", features = ["as-bytes"] }
```

Comment:
This is a breaking change, because `rgb` version `0.8.0` does not have the `as-bytes` feature.

It is generally a bad idea to truncate versions like that. Relaxed version dependencies in Cargo cause unexpected breakage, because Cargo often but not always interprets them as latest, so broken versions are apparent only to users who actually need to downgrade them. They wreck havoc when using `-Z minimal-versions`. It does less harm to always bump all versions to latest than to try to be flexible.

Reply:
This is being reverted (pretty much all of it except the function I added), please disregard all other changes :)